keps/sig-scheduling/1819-scheduler-extender/kep.yaml

```diff
@@ -0,0 +1,27 @@
+title: Scheduler Extender
+kep-number: 1819
+authors:
+  - "@ravigadde"
+owning-sig: sig-scheduling
+status: implemented
+creation-date: 2020-05-28
+reviewers:
+  - "@ahg-g"
+  - "@Huang-Wei"
+approvers:
+  - "@ahg-g"
+  - "@Huang-Wei"
+prr-approvers:
+  - "@wojtek-t"
+replaces:
+  - "https://github.com/cofyc/community/blob/master/contributors/design-proposals/scheduling/scheduler_extender.md"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
```

Comment:
I'm not sure which stage it should be. Promote it to GA when we moved to versioned API?

Reply:
SG. So that once we moved to versioned API, we start with v1beta1. (current v1 string doesn't make sense as it's not versioned)

Reply:
I think it is GA already.

Reply:
according to the [definitions described here](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#using-a-feature), it's GA because it can't be disabled now and feature gate is not needed.

Reply:
updated it to stable (GA) according to the definition, I think we can still make improvements no matter what stage it is